fastchat/serve/monitor/monitor.py

```diff
@@ -1099,10 +1187,10 @@ def build_demo(elo_results_file, leaderboard_table_file, arena_hard_leaderboard)
         button_small_text_size="20px",
         button_large_text_weight="100",
         button_small_text_weight="100",
-        button_shadow="*shadow_drop_lg",
-        button_shadow_hover="*shadow_drop_lg",
+        # button_shadow="*shadow_drop_lg",
```

Comment:
@yixin-huang1 Why did you comment these out?

Reply:
+1 why?